playbooks/host_vars/marist-1.yml

```diff
@@ -0,0 +1,5 @@
+---
+zowe_sanity_test_testcases: "./test/**/!(api-doc-gen.js)"
+zowe_token_name: ZWETOKEN
+zowe_token_label: jwtsecret
```

Comment:
It's better also define these 2 variables in `playbooks/roles/configure/defaults/main.yml` but with empty value.

Reply:
Sorry a second thought, I think we shouldn't leave them empty by default, but give a reasonable default name. Otherwise we will have to define the token name and label for all other hosts.